tools/ci-build/changelogger/src/new_entry.rs

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use anyhow::Context;
+use clap::Parser;
+use smithy_rs_tool_common::changelog::{FrontMatter, Markdown, Reference, Target};
+use smithy_rs_tool_common::git::find_git_repository_root;
+use smithy_rs_tool_common::here;
+use std::path::PathBuf;
+
+#[derive(Parser, Debug, Eq, PartialEq)]
+pub struct NewEntryArgs {
```

Comment:
I'd consider also adding short short flag versions for some of these to shorten the command line

Reply:
Provided short flags for commonly used options (provided `-b` for `bug-fix` rather than for `breaking` since `bug-fix` will be specified more frequently) in 83e67dc.